components/kyma-environment-broker/go.mod

```diff
@@ -78,6 +77,7 @@ require (
 	github.com/evanphx/json-patch v5.6.0+incompatible // indirect
 	github.com/felixge/httpsnoop v1.0.2 // indirect
 	github.com/fsnotify/fsnotify v1.5.1 // indirect
+	github.com/gardener/gardener v1.44.4 // indirect
```

Comment:
indirect but still present :-)

Reply:
yeah, next step is provisioner, that should be the last one
```
$ go mod graph | grep ' github.com/gardener/gardener'
github.com/kyma-project/control-plane/components/kyma-environment-broker github.com/gardener/gardener@v1.44.4
github.com/kyma-project/control-plane/components/provisioner@v0.0.0-20220420073630-1208b41756a8 github.com/gardener/gardener@v1.24.0
```
or even better, we convince gardener to split their API into it's own module, which would drastically improve depending on it.